src/api/app/mailers/event_mailer.rb

```diff
@@ -32,12 +33,6 @@ def event(subscribers, e)
 
     template_name = e.template_name
     orig = e.originator
-
-    # no need to tell user about this own actions
-    # TODO: make configurable?
-    subscribers.delete(orig)
```

Comment:
Yes, see 904879d9eb8e81e7d0d504d6ceeb0fd596b09832
```because it is already done in EventFindSubscriptions#filter_toconsider and add specs for it.```
Also I added a test in this commit, too!